### PR TITLE
[RAFlow][WIP]: Rentables - Is non recur charge flag

### PR DESCRIPTION
**Pull request by abataub**

### CODE REVIEW

File: webclient/html/formar.html [hunk 1]
```diff
@@ -46,6 +46,10 @@
         <label>Is SecDep ASM:</label>
         <div><input name="IsSecDepASM" type="checkbox" tabindex="8" /></div>
     </div>
+    <div class="w2ui-field">
+        <label>Is Non Recur Charge:</label>
+        <div><input name="IsNonRecurCharge" type="checkbox" tabindex="8" /></div>
```
Comment (sudip-aubergine) [inline webclient/html/formar.html]: please change the tabindex value.

File: webclient/js/elems/raflow/rentables.js [hunk 2]
```diff
@@ -742,7 +754,15 @@ window.loadRARentablesGrid = function () {
                                 var BID = getCurrentBID();
                                 app.raflow.arList[BID].forEach(function(item) {
                                     if (event.value_new.id == item.ARID) {
+                                        console.log(item);
```
Comment (sudip-aubergine) [inline webclient/js/elems/raflow/rentables.js]: remove/comment console.log if it's no longer used

File: webclient/js/elems/raflow/rentables.js [hunk 3]
```diff
@@ -742,7 +754,15 @@ window.loadRARentablesGrid = function () {
...
                                         f.record.Amount = item.DefaultAmount;
+
+                                        if(item.FLAGS === 66){
+                                            f.record.RentCycle = app.cycleFreq[0];
+                                        }
+
+                                        console.log(item.FLAGS);
```
Comment (sudip-aubergine) [inline webclient/js/elems/raflow/rentables.js]: remove/comment console.log if it's no longer used

File: webclient/html/formar.html (hunk 1 above)
Comment (abataub) [reply webclient/html/formar.html]: Okay.

File: webclient/js/elems/raflow/rentables.js (hunk 2 above)
Comment (abataub) [reply webclient/js/elems/raflow/rentables.js]: console.log will be removed in final merge in each pull request(It same for future PR).

File: webclient/js/elems/raflow/rentables.js (hunk 3 above)
Comment (abataub) [reply webclient/js/elems/raflow/rentables.js]: As It is in progress. It requires. It'll be removed at end of the feature development completed.

File: webclient/js/elems/raflow/rentables.js (hunk 2 above)
Comment (sudip-aubergine) [reply webclient/js/elems/raflow/rentables.js]: all right

File: webclient/js/elems/raflow/rentables.js (hunk 3 above)
Comment (sudip-aubergine) [reply webclient/js/elems/raflow/rentables.js]: all right

**Pull request merged**